Framework/PythonInterface/mantid/plots/_functions.py

```diff
@@ -0,0 +1,188 @@
+import numpy
```

Comment:
Can we add the GPL licence header and a module doc string?

Reply:
The module doc string is in the `__init__.py` file.